HISTORY.md

```diff
@@ -1,3 +1,15 @@
+## CONNECT 5.3 Release
+### Summary
+The 5.3 release includes these exciting new features:
+
+* Import chain of trust wizard
+* Configurable SHA version specification
+* Server Name Indication (SNI) support and proof of concept
+* Optimized SAML assertion builder
+* CONNECT wiki enhancement
+
+For more information refer to the [Release Notes]([https://connectopensource.atlassian.net/wiki/x/BQAgKg]).
```

Comment:
I don't see the hyperlink in here. Can you please verify in yr own repo? https://github.com/vdmehta06/CONNECT/blob/CONN-2483/HISTORY.md